code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -208,6 +216,11 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/silver_sulfadiazine/overdose_process(mob/living/M)
+	M.adjustFireLoss(1, 0)
```

Comment:
this is going to be canceled out by the healingn

look at how the trekchems do it